src/trackers/classes/crawl.js

```diff
@@ -131,6 +160,35 @@ function _getDomainSummaries (crawl) {
     return domainSummary
 }
 
+function _getFingerprintWeights (crawl) {
```

Comment:
can we extract that as a helper? it'd be easier to test.

Reply:
👍